test/parallel/test-child-process-fork-net.js

```diff
@@ -3,6 +3,7 @@ require('../common');
 const assert = require('assert');
 const fork = require('child_process').fork;
 const net = require('net');
+const common = require('../common');
```

Comment:
`common` is already required a few lines up. It just needs to be assigned to a variable.